demisto_sdk/commands/validate/file_validator.py

```diff
@@ -714,6 +733,9 @@ def is_valid_structure(self):
         if self.validate_all:
             self.validate_all_files()
             return self._is_valid
+        if self.pack_path:
```

Comment:
once you remove the additional flag, this should be moved into `if self.file_path:`

Reply:
What do you mean here?

Reply:
it you look at line `if self.file_path:`, thats where we actually use the user input of file to validate, so this should be moved there